### ISSUE TITLE
Yara cannot scan chinese filename?

### ISSUE THREAD

**Issue by cheapylam** [VirusTotal/yara#1487]
Hi, I am new to Yara.
I am using Yara to perform scanning on malicious web shell script using core.webshell_detection.yara provided by NSA/ASD Mitigating Web Shells

I ran from windows command line and the target file with Chinese filename and got the following error.
error scanning {filename} could not open file

I am using windows 10 with English char set machine. What went wrong here? Because as much as I see this, it is very severe issue, because someone can create a malicious file with this filename and walk through undetected. Am I right?

![擷取](https://user-images.githubusercontent.com/33109683/116866042-4ecd6c00-ac3d-11eb-8494-4913ff0c2184.PNG)

**Comment by plusvic** [comment]
This should be fixed after https://github.com/VirusTotal/yara/pull/1491. Could try with the latest version in the master branch and let me know if it works fine?

**Comment by cheapylam** [comment]
May I know how to generate an executable file [yara.exe] from the master branch?

**Comment by cheapylam** [comment]
@plusvic 
Sorry, may I know the new version 4.1.1 is it including the fixed #1491 ?

Please also note that the new version 4.1.1  ==> yara.exe -v <=== version not updated
![yara_4 1 1_ver_not_update](https://user-images.githubusercontent.com/33109683/119428888-4fd75200-bd40-11eb-8a16-b1b474aeb0e3.PNG)

**Comment by plusvic** [comment]
No, version 4.1.1 is a minor update, including only bug fixes. The unicode support will be released in version 4.2.0.

**Comment by cheapylam** [comment]
Thank you :D
